lib/jsonapi_suite.rb

```diff
@@ -5,6 +5,8 @@
 require 'jsonapi_errorable'
 
 require "jsonapi_suite/version"
+require "jsonapi_suite/strong_resources/exception_handler"
+require "jsonapi_suite/strong_resources/controller_mixin"
```

Comment:
My main comment is I found the naming here really confusing until I dug into it. I'm thinking exception_handler can move to SR itself, and the mixin behavior could move into the original SR controller mixin? Any issues you see with that?